java/ql/src/semmle/code/java/frameworks/javase/URL.qll

```diff
@@ -0,0 +1,47 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+/* Am URL constructor expression */
+class UrlConstructor extends ClassInstanceExpr {
```

Comment:
Let's move all the contents of this file to `Networking.qll` - that already contains `TypeUrl`.